pagination/keysetpagination/header.go

```diff
@@ -104,7 +105,7 @@ func Parse(q url.Values, p PageTokenConstructor) ([]Option, error) {
 		}
 		opts = append(opts, WithToken(parsed))
 	}
-	if q.Has("page_size") {
+	if q.Get("page_size") != "" {
```

Comment:
Why this? :thinking:

Reply:
I've encountered `GET /admin/identities?page_size=` a bunch of times while working on this. I can also see an SDK generating code like that, and we should be prepared for that. Previously, this would return 400 with an internal error message referencing `strconv.Atoi`. Not it's just equal to default.